test/buffer-source.js

```diff
@@ -81,16 +81,23 @@ const bufferSourceConstructors = [
 
 const bufferSourceCreators = [];
 for (const constructor of bufferSourceConstructors) {
+    const { name } = constructor;
     bufferSourceCreators.push(
         {
-            typeName: constructor.name,
-            label: `${constructor.name} same realm`,
+            typeName: name,
+            label: `${name} same realm`,
             creator: () => new constructor(new ArrayBuffer(0))
         },
         {
-            typeName: constructor.name,
-            label: `${constructor.name} different realm`,
-            creator: () => vm.runInContext(`new ${constructor.name}(new ArrayBuffer(0))`, differentRealm)
+            typeName: name,
+            label: `${name} different realm`,
+            creator: () => vm.runInContext(`new ${name}(new ArrayBuffer(0))`, differentRealm)
+        },
+        {
+            typeName: `forged ${name}`,
```

Comment:
Can we keep `typeName` the original `constructor.name`, just like all the other test cases?

Reply:
I originally did this before adding `isForged`, but then I forgot to change this back.